rpcd/playbooks/roles/kibana/tasks/kibana_pre_install.yml

```diff
@@ -13,50 +13,40 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-- name: Install Kibana apt packages
-  apt:
-    pkg: "{{ item }}"
-    state: latest
-    update_cache: yes
-    cache_valid_time: 600
-  register: install_apt_packages
-  until: install_apt_packages|success
+- name: Add kibana apt-keys
+  apt_key:
+    url: "{{ item.url }}"
+    state: "{{ item.state }}"
+  with_items: kibana_apt_keys
+  when: kibana_apt_keys is defined
+  register: add_kibana_keys_url
+  until: add_kibana_keys_url|success
```

Comment:
hm.. i somehow lost the item below.

The retry pattern is on every 'add apt-keys' and 'add apt-repo' action in every role in the repository.